### DISCUSSION TITLE
a

### DISCUSSION THREAD

**Discussion by hell001** [LeoNatan/LNPopupController#469] [General] [answered]
a

**Comment by iDevelopper** [comment, marked as answer]
> Or even remove that entirely and let the user control the swipe the entire way down?
> Also wondering the same thing for presenting/swiping up

You can set popupInteractionStyle property of the container to LNPopupInteractionStyle.drag.

**Reply by LeoNatan** [reply]
This is documented, and even presented as a gif in the readme. Please read the docs @hell001
